Adding example of item access operator `::` (#112)
microsoft/qsharp-language: Specifications/Language/3_Expressions/ItemAccessExpressions.md

    let im = complex::Imaginary;  // item access via name
```

> The item access operator (`::`) retrieves named items. For example, it can be used to unpack a newtype operation parameter in a neater way than with individual item extraction: 

Reviewer: ```suggestion
The item access operator (`::`) retrieves named items, as illustrated by the following example:
```
Editing for correctness.

```qsharp

> newtype TwoBitString = (bit1: String, bit2: String);
> 
> operation Str(bits: TwoBitString):String {
>     let b1= bits::bit1;
>     let b2 = bits::bit2;
>     return b1+b2;
> }

Reviewer 2: Some Q# style guide adherence nits:
```suggestion
newtype TwoBitString = (bit1: String, bit2: String);

operation Str(bits : TwoBitString) : String {
    let b1 = bits::bit1;
    let b2 = bits::bit2;
    return b1 + b2;
}
```
I'd also suggest either making `Str` a function, or renaming it to be consistent with operation naming conventions (that is, as a verb). The last thing I'd raise is that it's not clear to me what constrains the named items of `TwoBitString` to representing bit strings; perhaps another name may be more clear for this example?

Author: Thank you so much for your comments @cgranade! 
I will rename the function and the NewType to clarify them :)
